server/migrator/Schema.java

```diff
@@ -59,20 +59,17 @@ public String getSchema() {
              final Grakn.Transaction tx = session.transaction(Arguments.Transaction.Type.READ)) {
             tx.concepts()
                     .getRootAttributeType()
-                    .getSubtypes()
-                    .filter(x -> !x.isRoot())
+                    .getSubtypesDirect()
```

Comment:
I like that this method has resulted in the removal of `filter(x -> !x.isRoot())` 👍

Reply:
Cool.